README.md

```diff
@@ -1,101 +1,58 @@
-
-## Contributing
-
-Don't hesitate to create a pull request.
-Every contribution is appreciated.
-
-### Notes for contributors
+```sh
+make serve-dev
+```
 
-* Don't make changes on the main repo, do them in your fork.
-* Every sentence should be on a new line.
-* Test locally before submitting a PR.
-* Always submit a PR before merging.
-* The author of the PR should merge unless they cannot and it's time-sensitive.
+If you would like to see the production view (without drafts and with minified CSS), run:
 
-1. Fork the official repo into your personal GitHub.
-2. The new forked copy is cloned to your local system.
-3. Git remote path for the 'official' repository is added to the local clone: ```git remote add upstream git@github.com:viamrobotics/docs.git```
+```sh
+make serve-prod
+```
 
-When you are ready to contribute changes to the docs:
+You can also run `hugo serve` after installing Hugo to show the production view.
 
-1. Make sure you are on the main branch: ```git switch main```
-2. Sync your forked main branch with the official repo: ```git pull upstream main```
-3. Create a new branch for your changes: ```git switch -c my-new-feature```
-4. Edit some docs...
-5. Commit your changes: ```git commit -am 'Add some feature'```
-6. Make sure your local branch is still up-to-date with the official repo: ```git pull upstream main```
-7. Push to the branch: ````git push origin my-new-feature````
-8. Submit a pull request :D
+### Generate HTML docs
 
-### Converting from Google Docs to Markdown
+To generate the full HTML version of the docs run:
 
-1. Export your Google Document as `.docx` file.
-   Go to "File" -> "Download as" -> "Microsoft Word (.docx)".
-2. Use [Pandoc](https://pandoc.org/) to convert to markdown and export media to a directory.
-   Here's an example command:
+```sh
+make build-prod
+```
 
-   ```bash
-   pandoc --wrap=preserve --extract-media ./ NAME_OF_YOUR_DOCUMENT.docx -o NAME_OF_YOUR_DOCUMENT.md
-   ```
+You can serve the resulting docs with:
 
-3. After converting your document to markdown, you will still need to manually fix any conversion errors.
-   Here are some things that you will need to address:
-   1. Manually split up sentences so there is only one sentence per line.
-   2. Update the path name on all images to point to the correct directory.
-   3. Convert all internal links to relative links.
-   4. Run the markdown through the linter to ensure that it is formatted correctly.
-4. be sure that you test locally to ensure that the document looks correct and submit a PR.
+```sh
+python3 -m http.server 9000 --directory public
+```
 
-## Important Things about Hugo
+## Publishing
 
-### Linting
+The docs are automatically published when a PR merges.
 
-We are using `markdownlint` to ensure consistency in our documentation.
-Be sure that you have the [markdownlint VS Code extension](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) installed locally.
+## Hugo Quickstart
```

Comment:
It still seems like all of this could go in contributing?

Reply:
it could but that would make the contributing page very long - and that's what gets shown to everyone making a contribution. So I think linking to this if it's needed might be nicer.